virttest/test_setup.py

```diff
@@ -1496,7 +1496,7 @@ def sr_iov_setup(self):
         # Re-probe driver with proper number of VFs once more and raise
         # exception
         if re_probe:
-            if not self.remove_driver() and not self.modprobe_driver():
+            if not self.remove_driver() or not self.modprobe_driver():
```

Comment:
`or` should be removed. when reprobe have to be done, clean up the VF and remove driver if any, and perform a probe once before raising.

Reply:
remove_driver() and modprobe_driver() return True on success.  If we keep `and` here,  when remove_driver() success,  modprobe_driver() will not be executed. That's why I use `or` instead. Thanks!

Reply:
Yes,  agree with @huangyum ,  `or` is the correct logic i think. @balamuruhans